sirtypes.h

```diff
@@ -77,49 +77,303 @@ typedef enum {
+    SIRTC_147      = 147,
+    SIRTC_148      = 148,
+    SIRTC_149      = 149,
+    SIRTC_150      = 150,
+    SIRTC_151      = 151,
+    SIRTC_152      = 152,
+    SIRTC_153      = 153,
+    SIRTC_154      = 154,
+    SIRTC_155      = 155,
+    SIRTC_156      = 156,
+    SIRTC_157      = 157,
+    SIRTC_158      = 158,
+    SIRTC_159      = 159,
+    SIRTC_160      = 160,
+    SIRTC_161      = 161,
+    SIRTC_162      = 162,
+    SIRTC_163      = 163,
+    SIRTC_164      = 164,
+    SIRTC_165      = 165,
+    SIRTC_166      = 166,
+    SIRTC_167      = 167,
+    SIRTC_168      = 168,
+    SIRTC_169      = 169,
+    SIRTC_170      = 170,
+    SIRTC_171      = 171,
+    SIRTC_172      = 172,
+    SIRTC_173      = 173,
+    SIRTC_174      = 174,
+    SIRTC_175      = 175,
+    SIRTC_176      = 176,
+    SIRTC_177      = 177,
+    SIRTC_178      = 178,
+    SIRTC_179      = 179,
+    SIRTC_180      = 180,
+    SIRTC_181      = 181,
+    SIRTC_182      = 182,
+    SIRTC_183      = 183,
+    SIRTC_184      = 184,
+    SIRTC_185      = 185,
+    SIRTC_186      = 186,
+    SIRTC_187      = 187,
+    SIRTC_188      = 188,
+    SIRTC_189      = 189,
+    SIRTC_190      = 190,
+    SIRTC_191      = 191,
+    SIRTC_192      = 192,
+    SIRTC_193      = 193,
+    SIRTC_194      = 194,
+    SIRTC_195      = 195,
+    SIRTC_196      = 196,
+    SIRTC_197      = 197,
+    SIRTC_198      = 198,
+    SIRTC_199      = 199,
+    SIRTC_200      = 200,
+    SIRTC_201      = 201,
+    SIRTC_202      = 202,
+    SIRTC_203      = 203,
+    SIRTC_204      = 204,
+    SIRTC_205      = 205,
+    SIRTC_206      = 206,
+    SIRTC_207      = 207,
+    SIRTC_208      = 208,
+    SIRTC_209      = 209,
+    SIRTC_210      = 210,
+    SIRTC_211      = 211,
+    SIRTC_212      = 212,
+    SIRTC_213      = 213,
+    SIRTC_214      = 214,
+    SIRTC_215      = 215,
+    SIRTC_216      = 216,
+    SIRTC_217      = 217,
+    SIRTC_218      = 218,
+    SIRTC_219      = 219,
+    SIRTC_220      = 220,
+    SIRTC_221      = 221,
+    SIRTC_222      = 222,
+    SIRTC_223      = 223,
+    SIRTC_224      = 224,
+    SIRTC_225      = 225,
+    SIRTC_226      = 226,
+    SIRTC_227      = 227,
+    SIRTC_228      = 228,
+    SIRTC_229      = 229,
+    SIRTC_230      = 230,
+    SIRTC_231      = 231,
+    SIRTC_232      = 232, /**< Greyscale begins here, through 255. */
+    SIRTC_233      = 233,
+    SIRTC_234      = 234,
+    SIRTC_235      = 235,
+    SIRTC_236      = 236,
+    SIRTC_237      = 237,
+    SIRTC_238      = 238,
+    SIRTC_239      = 239,
+    SIRTC_240      = 240,
+    SIRTC_241      = 241,
+    SIRTC_242      = 242,
+    SIRTC_243      = 243,
+    SIRTC_244      = 244,
+    SIRTC_245      = 245,
+    SIRTC_246      = 246,
+    SIRTC_247      = 247,
+    SIRTC_248      = 248,
+    SIRTC_249      = 249,
+    SIRTC_250      = 250,
+    SIRTC_251      = 251,
+    SIRTC_252      = 252,
+    SIRTC_253      = 253,
+    SIRTC_254      = 254,
+    SIRTC_255      = 255,
```

Comment:
And this is kind of ridiculous, too. I don't think there needs to be an enum entry for every color from 17 to 255. It should just have a comment saying use the decimal form. We only need the last two.